collectors/apps.plugin/apps_groups.conf

```diff
@@ -216,26 +223,30 @@ print: cups* lpd lpq
 # -----------------------------------------------------------------------------
 # time servers and clients
 
-time: ntp* systemd-timesyn* chronyd
+time: ntp* systemd-timesyn* chronyd ptp*
 
 # -----------------------------------------------------------------------------
 # dhcp servers and clients
 
-dhcp: *dhcp*
+dhcp: *dhcp* dhclient
 
 # -----------------------------------------------------------------------------
 # name servers and clients
 
-dns: named unbound nsd pdns_server knotd gdnsd yadifad dnsmasq systemd-resolve* pihole*
+dns: named unbound nsd pdns_server knotd gdnsd yadifad dnsmasq systemd-resolve* pihole* avahi-daemon avahi-dnsconfd
 dnsdist: dnsdist
 
 # -----------------------------------------------------------------------------
 # installation / compilation / debugging
 
 build: cc1 cc1plus as gcc* cppcheck ld make cmake automake autoconf autoreconf
-build: cargo rustc bazel buck git gdb valgrind*
+build: cargo rustc bazel buck git gdb valgrind* rpmbuild dpkg-buildpackage
 
-apt: apt*
+# -----------------------------------------------------------------------------
+# package management
+
+packagemanager: apt* dpkg* dselect dnf yum rpm zypp* yast* pacman xbps* swupd* emerge*
```

Comment:
@Ferroin You forgot `slackpkg`. :cry:

Reply:
@Ferroin please don't add. Let @thiagoftsm handle it.

Reply:
Considering @ilyam8's comment, I also think you could remove `pacman` and let him to add this. :smile: